assets/js/vsipCalendar.js

```diff
@@ -0,0 +1,46 @@
+const calendarDayClass = 'vsip-calendar__day'
+const calendarDaySelectedClass = 'vsip-calendar__day--selected'
+const calendarDayGroupClass = 'vsip-calendar__day-group'
+const calendarDayGroupActiveClass = 'vsip-calendar__day-group--active'
+
+function handleSelectDate(event) {
+  event.preventDefault()
+  const dateToShow = event.target.parentElement.dataset.date
+
+  // remove highlighted day
+  document.querySelector(`.${calendarDaySelectedClass}`).classList.remove(calendarDaySelectedClass)
+  
+  // hide active form group
+  document.querySelector(`.${calendarDayGroupActiveClass}`).classList.remove(calendarDayGroupActiveClass)
+
+  // highlight selected day
+  document.querySelector(`.${calendarDayClass}[data-date='${dateToShow}']`).classList.add(calendarDaySelectedClass)
+
+  // show selected day's slots
+  const selectedFormGroup = document.querySelector(`.${calendarDayGroupClass}[data-date='${dateToShow}']`)
+  selectedFormGroup.classList.add(calendarDayGroupActiveClass)
+
+  // scroll to slots if first not in viewport
```

Comment:
Experimenting with scrolling user down automatically - may remove this

Reply:
Do we need an aria attribute to explain what's happening or is it not relevant here?

Reply:
Yes, quite possibly. I was looking into `aria-live` to try and understand that.... However, Phoebe isn't keen on it scrolling down automatically so she's going to mull it over. Once we've got the behaviour agreed I want to revisit whether we've got all the arias we need